test/cmd/admin.sh

```diff
@@ -16,8 +16,7 @@ trap os::test::junit::reconcile_output EXIT
   oc delete users/orphaned-user
   oc delete identities/anypassword:orphaned-user
   oc delete identities/anypassword:cascaded-user
-  oc adm policy reconcile-cluster-roles --confirm --additive-only=false
-  oc adm policy reconcile-cluster-role-bindings --confirm --additive-only=false
+  oc adm create-bootstrap-policy-file --filename >( cat ) | oc auth reconcile --remove-extra-permissions --remove-extra-subjects -f -
```

Comment:
`oc adm create-bootstrap-policy-file` is on my naughty list too.  It can't logically live in the client.  At the start of the test, get the current list of resources